docs/source/model_result.rst

```diff
@@ -0,0 +1,27 @@
+Result
```

Comment:
how soon is this shifting to `SingleResult`? worth doing `Result (SingleResult)` and then reversing after the shift?

Reply:
Let's put the name up for a vote then switch.